README.md

```diff
@@ -212,7 +212,7 @@ There are JS Buy SDK specific example apps in Node, Ember, and React. You can us
 
 ## Documentation
 
-For full API documentation go check out the [API docs](https://shopify.github.io/js-buy-sdk).
+For full API documentation go check out the [API docs](https://shopify.github.io/js-buy-sdk). You can also view our [Changelog](https://github.com/Shopify/js-buy-sdk/CHANGELOG.md).
```

Comment:
I would maybe put this at the top under its own heading if it's something you plan on updating frequently.